server/game/mode/fugitive/ServerFugitiveGame.gd

```diff
@@ -41,7 +41,7 @@ func _ready():
 
 
 func load_map():
-	.load_map()
+	.load_map()	
```

Comment:
clean up

Reply:
Oh, I don't remember changing this file, good stuff!